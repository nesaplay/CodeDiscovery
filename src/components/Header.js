import React from 'react'
import { Header, Icon, Grid, Divider } from 'semantic-ui-react'
import { Link } from 'react-router-dom'

export default () => {
	const { Row, Column } = Grid

	return (
		<header>
			<Grid>
				<Row style={{ marginTop: '20px' }}>
					<Column>
						<Link to="/">
							<Header as="h3" icon textAlign="center">
								<Icon name="users" circular />
								<Header.Content>Code Discovery</Header.Content>
							</Header>
						</Link>
						<Divider />
					</Column>
				</Row>
			</Grid>
		</header>
	)
}
