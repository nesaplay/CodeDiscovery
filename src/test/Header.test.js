import React from 'react'
import Header from '../components/Header'

it('renders a component', () => {
	const wrapper = shallow(
        <Header />
    )

	expect(wrapper).toMatchSnapshot()
})
