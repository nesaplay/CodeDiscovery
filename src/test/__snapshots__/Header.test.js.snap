// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a component 1`] = `
<header>
  <Grid>
    <GridRow
      style={
        Object {
          "marginTop": "20px",
        }
      }
    >
      <GridColumn>
        <Link
          replace={false}
          to="/"
        >
          <Header
            as="h3"
            icon={true}
            textAlign="center"
          >
            <Icon
              as="i"
              circular={true}
              name="users"
            />
            <HeaderContent>
              Code Discovery
            </HeaderContent>
          </Header>
        </Link>
        <Divider />
      </GridColumn>
    </GridRow>
  </Grid>
</header>
`;
